atintegrators/BendLinearPass.c

```diff
@@ -10,7 +10,7 @@
 #include "atphyslib.c"
 
 
-#define SQR(X) X*X

```

Comment:
Just for my culture, could you explain this modification?

Reply:
@swhite: to avoid repeating 10 times the definition of SQR, it has been included in `atelem.c`, but in its correct form:
```C
#define SQR(X) ((X)*(X))
```
Keeping a duplicate definition in existing modules is allowed, provided it's exactly the same. But this one was different, so I had to remove it. Other existing files are unchanged.

Reply:
Right, but why are the parenthesis needed?

Reply:
without parentheses:

`3*SQR(a+b)` -> `3*a+b*a+b`: completely wrong

You always need to parenthesise everything when defining macros.